can/interfaces/pcan/basic.py

```diff
@@ -498,9 +500,16 @@ class PCANBasic:
     """
 
     def __init__(self):
-        # Loads the PCANBasic.dll
+        # Loads the PCANBasic.dll and checks if driver is available
         if platform.system() == "Windows":
             self.__m_dllBasic = windll.LoadLibrary("PCANBasic")
+            aReg = winreg.ConnectRegistry(None, winreg.HKEY_LOCAL_MACHINE)
+            try:
+                aKey = winreg.OpenKey(aReg, r"SOFTWARE\PEAK-System\PEAK-Drivers")
+                winreg.CloseKey(aKey)
+            except WindowsError:
+                logger.error("Exception: The PEAK-driver couldn't be found!")
+            winreg.CloseKey(aReg)
```

Comment:
This might be done in a finally block.

Reply:
will have a look aswell here

Reply:
See commit https://github.com/hardbyte/python-can/pull/724/commits/f542d8ce3a731521f862d72ab7a4a793a69c7902
Is this what you meant? Didn't see how else I'd use a finally block here

Reply:
Yeah, just move the `winreg.CloseKey(aReg)` in line 512 into the finally block and then also remove the identical call in line 509.

Reply:
@felixdivo line 509 is different from the line 512 one. I almost also made the same assumption but the former is closing the key, the latter is closing the registry.

Reply:
Whoops, yep. Then this is fine. Thanks for taking the time.